Microsoft.Toolkit.Uwp.Connectivity/Network/ConnectionInformation.cs

```diff
@@ -135,9 +135,9 @@ public bool IsInternetOnMeteredConnection
         public byte? SignalStrength { get; private set; }
 
         /// <summary>
-        /// Gets signal strength for the current Internet Connection Profile.
+        /// Gets the network names associated with the current Internet Connection Profile.
         /// </summary>
-        /// <returns>value of <see cref="NetworkConnectivityLevel"/></returns>
+        /// <returns>value of <see cref="IReadOnlyList<string>"/></returns>
```

Comment:
Oh does this need to be escaped or something? Looks like something is causing a build error:

```
D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Network\ConnectionInformation.cs(140,55): error CS1570: XML comment has badly formed XML -- '{0}' [D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Microsoft.Toolkit.Uwp.Connectivity.csproj]
         D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Network\ConnectionInformation.cs(140,55): error CS1570: XML comment has badly formed XML -- 'see' [D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Microsoft.Toolkit.Uwp.Connectivity.csproj]
         D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Network\ConnectionInformation.cs(140,68): error CS1570: XML comment has badly formed XML -- 'returns' [D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Microsoft.Toolkit.Uwp.Connectivity.csproj]
         D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Network\ConnectionInformation.cs(141,1): error CS1570: XML comment has badly formed XML -- 'returns' [D:\a\1\s\Microsoft.Toolkit.Uwp.Connectivity\Microsoft.Toolkit.Uwp.Connectivity.csproj]
```

Reply:
Ops, my bad. I did the change in github.
```suggestion
        /// <returns>value of <see cref="IReadOnlyList{string}"/></returns>
```